services/kafka/config.go

```diff
@@ -44,6 +44,12 @@ type Config struct {
 	InsecureSkipVerify bool `toml:"insecure-skip-verify" override:"insecure-skip-verify"`
 }
 
+func NewConfig() Config {
+	return Config{
+		Enabled: false,
+	}
+}
```

Comment:
Why do we want a default enabled kakfa config?

Reply:
Do you mean, why do we want a default disabled kafka config?

Reply:
To be sure, it's `Enabled: false`. I could also just do `return Config{}`, I suppose!

Reply:
yes, just return an empty `Config{}`... and apparently I can't read :(

thanks

Reply:
No worries! Sounds good -- done.